attack/scripts/bashrc

```diff
@@ -122,17 +122,17 @@ starting_point() {
 
   #Determine if mode is node.
   if [[ "$MODE" == "node" ]]; then
-    NODE_TYPE="$(echo "${task_json}" | jq -r --arg TASK_NO "${task_no}" '.tasks | .[$TASK_NO] | ."starting-point"."node-id"')"
+    NODE_TYPE="$(echo "${task_json}" | jq -r --arg TASK_NO "${task_no}" '.tasks | .[$TASK_NO] | .startingPoint.nodeId')"
 
     if [[ "$NODE_TYPE" =~ "master" ]]; then
-      NODE_NUMBER="$(echo "$NODE_TYPE" | sed "s/node-id: 'master-//" | sed "s/'//")"
+      NODE_NUMBER="$(echo "$NODE_TYPE" | sed "s/nodeId: 'master-//" | sed "s/'//")"
 
       ssh_master "$NODE_NUMBER"
       return
     fi
 
     if [[ "$NODE_TYPE" =~ "node" ]]; then
-      NODE_NUMBER="$(echo "$NODE_TYPE" | sed "s/node-id: 'node-//" | sed "s/'//")"
+      NODE_NUMBER="$(echo "$NODE_TYPE" | sed "s/nodeId: 'node-//" | sed "s/'//")"
```

Comment:
Not your implementation, but why the double `sed`?  could we either:

* extract a sed script that could be invoked once
* or better change the task.yaml spec so that we dont need to do text manipulation at all - just select things with jq

Reply:
I realise the second option is quite a bit more work, but it *would* be much nicer IMO :)

Reply:
We already select the value with `jq`, this is a trim operation to get the number. I've updated to use bash parameter substitution.